src/Examples/Flowsheets/HDA_flowsheet_with_costing.ipynb

```diff
@@ -0,0 +1,1598 @@
+      "2022-07-21 10:57:27 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.D101.stripping_section[10].e_mole_frac_liq_out[0.0,toluene]\n",
+      "2022-07-21 10:57:27 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.D101.reboiler.control_volume.properties_in[0.0].mole_frac_comp[toluene]\n",
+      "2022-07-21 10:57:27 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.D101.stripping_section[10].properties_out[0.0].temperature\n",
+      "2022-07-21 10:57:27 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.D101.reboiler.control_volume.properties_in[0.0].temperature\n",
+      "2022-07-21 10:57:27 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.D101.stripping_section[10].properties_out[0.0].pressure\n",
+      "2022-07-21 10:57:27 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.D101.reboiler.control_volume.properties_in[0.0].pressure\n",
+      "2022-07-21 10:57:27 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.D101.reboiler.e_vap_flow[0.0]\n",
+      "2022-07-21 10:57:27 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.D101.stripping_section[10].properties_in_vap[0.0].flow_mol\n",
+      "2022-07-21 10:57:27 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.D101.reboiler.e_vap_frac[0.0,benzene]\n",
+      "2022-07-21 10:57:27 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.D101.stripping_section[10].properties_in_vap[0.0].mole_frac_comp[benzene]\n",
+      "2022-07-21 10:57:27 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.D101.reboiler.e_vap_frac[0.0,toluene]\n",
+      "2022-07-21 10:57:27 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.D101.stripping_section[10].properties_in_vap[0.0].mole_frac_comp[toluene]\n",
+      "2022-07-21 10:57:27 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.D101.reboiler.control_volume.properties_out[0.0].temperature\n",
+      "2022-07-21 10:57:27 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.D101.stripping_section[10].properties_in_vap[0.0].temperature\n",
+      "2022-07-21 10:57:27 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.D101.reboiler.control_volume.properties_out[0.0].pressure\n",
+      "2022-07-21 10:57:27 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.D101.stripping_section[10].properties_in_vap[0.0].pressure\n",
+      "Complete.\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# alias 'n' for hda model with distillation column\n",
+    "n = hda_with_distillation(tee=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## IDAES Generic Costing Framework\n",
+    "IDAES provides a rigorous costing package for process economics calculations, based on methods from the following source:\n",
```

Comment:
Again ,drop the "rigorous". I would say something like:

The IDAES Integrated Platform provides a library of capital costing correlations based on those in..."